driver-examples/src/main/java/com/datastax/driver/examples/basic/CreateAndPopulateKeyspace.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.driver.examples.basic;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.cql.Row;
+
+/**
+ * Creates a keyspace and tables, and loads some data into them.
+ *
+ * <p>Preconditions: - a Cassandra session is running and accessible through the contacts points
+ * identified by basic.contact-points (see application.conf)
+ *
+ * <p>Side effects: - creates a new keyspace "simplex" in the session. If a keyspace with this name
```

Comment:
All the side effects in all examples have their initial formatting altered. You need to replace the original dashes with proper HTML markup, otherwise the formatting plugin that we use will garble everything:

```
 * <p>Side effects:
 *
 * <ul>
 *   <li>creates a new keyspace "simplex" in the session. If a keyspace with this name * already
 *       exists, it will be reused;
 *   <li>creates two tables "simplex.songs" and "simplex.playlists". If they exist already, they
 *       will * be reused;
 *   <li>inserts a row in each table.
 * </ul>
```